internal/shared/util/image/cache.go

```diff
@@ -29,6 +29,7 @@ type LayerData struct {
 }
 
 type Cache interface {
+	ExtendCache
```

Comment:
Is that valid for any or only Helm
Would be better we rename it like ChartCache, something like?

Reply:
Only for Helm charts at this time. We would need to  see if it is possible to refactor the `Store()` method in the Cache interface to be able to store both Helm charts and OCI images to the cache.